iterations_runner.lua

```diff
@@ -40,6 +40,34 @@ local ffi = require("ffi")
 local json = require("json_nojit")
 local jitstats
 
+function add_luapackage_path(basepath, highpriority)
```

Comment:
Rather than `highpriority` can we call it something like `addtoend` or `addtostart` or something? And let's document that it's a boolean above the function.

Reply:
i didn't know what to really call it that wasn't really long winded or provided more guarantee than what it does.

Reply:
Let's go with `addtostart` then. `highpriority` makes promises that it can't quite guarantee to keep. Let's add a docstring (or whatever the Lua equivalent is) to the function so that we document what's going on.

Reply:
@ltratt I had just added it as plain comment to the function header

Reply:
Is that the standard Lua way of doing things?

Reply:
@ltratt well there is a thirdparty system called LuaDoc that is more the structured kind that you might be thinking of